docs/MIGRATING.md

```diff
@@ -0,0 +1,39 @@
+
+> n.b. "GitHub Actions" refers both to the CI service as a whole and individual components (lower-case "actions") that can be incorporated into your CI job, as described below.
+> 
+### Comparison of Travis CI and GitHub Actions features
+
+- GitHub Actions allows you to re-use bundled features ("actions") provided by GitHub and the community. Actions are essentially mini-applications that help accomplish common CI use cases such as checking out code, installing PHP, or creating a GitHub release based on the outcome of a job.
+- Both services execute jobs based on definitions provided via a YAML file. Travis CI reads `.travis.yml` from a repository root directory, whereas GitHub Actions reads YAML files in the `.github/workflows` directory.
+- Travis CI requires a paid subscription, though it does provide an open-source tier with certain restrictions. GitHub Actions is free and fully functional for all open-source projects.
+- Both services can build on Windows, macOS and Linux.
+- Both services support running jobs on pull requests, commits, and cron.
+- GitHub Actions provides a more robust security model, including API tokens that are generated on a per-job basis, tokens that are read-only for pull requests from forks, and the option to require manual approval prior to job execution.
+- Travis CI executes jobs in a rigidly-defined series of "stages", whereas GitHub Actions allows defining an arbitrary number of "steps".
+- Both services support build matrices, which dynamically create jobs based on combinations of job variables. However, whereas Travis CI created a build matrix directly from first-class primitives such as PHP versions, GitHub Actions requires defining an intermediary set of [matrix variables](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix). While this is generally more flexible than Travis CI build matrices, it also requires a little more YAML boilerplate.
+
+### Migrating an ORCA project from Travis CI to GitHub Actions
+
+ORCA provides self-documented template YAML files for Travis CI and GitHub Actions via its [example project](https://github.com/acquia/orca/tree/develop/example). The recommended process to migrate from Travis CI to GitHub Actions is to:
+
+1. Compare your project's `.travis.yml` to the [stock template](https://github.com/acquia/orca/blob/develop/example/.travis.yml) and document any differences (i.e. customizations). Especially, note environment variables such as `ORCA_SUT_NAME` and `ORCA_SUT_BRANCH`.
+2. Remove `.travis.yml` from your project and copy ORCA's [GitHub Actions template workflow](https://github.com/acquia/orca/tree/develop/example/.github/workflows) to the root directory.
+3. Apply to your GitHub Actions workflow any customizations documented in step 1.
```

Comment:
```suggestion
1. Compare your project's `.travis.yml` to the [stock template](https://github.com/acquia/orca/blob/develop/example/.travis.yml) and document any differences (i.e., customizations). Especially, note environment variables such as `ORCA_SUT_NAME` and `ORCA_SUT_BRANCH`.
1. Remove `.travis.yml` from your project and copy ORCA's [GitHub Actions template workflow](https://github.com/acquia/orca/tree/develop/example/.github/workflows) to the root directory.
1. Apply to your GitHub Actions workflow any customizations documented in step 1.
```

As Drupal core contribution taught me, ["i.e." and "e.g." are always followed by a comma](https://jakubmarian.com/comma-after-i-e-and-e-g/#:~:text=Virtually%20all%20American%20style%20guides,motherboards%2C%20graphic%20cards%2C%20CPUs.). 🙂 

In Markdown ordered lists, I begin every line "1." to reduce diff noise when a line added to or removed from the middle.